nmurillon/FCManager-front
# [config] Set up the project

# What is this PR about ?

This PR adds the basic config to run a Nextjs project

# Features list

- Closes #1 

# How to test ?

- Run `npm i` and `npm run dev`
- Go to [you localhost](http://localhost:3000/)
- The server should be working
